source/getting-started/faq.rst

```diff
@@ -574,3 +583,45 @@ ___________________________________________________________________
 
 The warning message is the result of reassigning a floating IP to a new instance, and is a normal security precaution built into SSH.
 To learn how to address this issue, please see :ref:`connecting-via-ssh`.
+
+.. _faq-experiment-precis:
+
+_____________________________
+Experiment Precis
+_____________________________
+
+.. _what-is-experiment-precis:
+
+What is Experiment Precis?
+____________________________
+
+Just like the Linux ``history`` command that you use for figuring out what you did to solve a problem, Chameleon testbed provides a similar functionality to help you record your experiment setups on Chameleon, so that it becomes easier for you to make your experiment reproducible.
```

Comment:
```suggestion
Just like the Linux ``history`` command that you use for figuring out what you did to solve a problem, the Chameleon testbed provides a similar functionality to help you record your experiment setups on Chameleon, so that it becomes easier for you to make your experiment reproducible.
```